### PR TITLE
update the vaccination copy and feedback link

### PR DESCRIPTION
**Pull request by pnavarrc**
Update the copy of the vaccination block to match the new copy (see the [Mocks](https://www.figma.com/file/8BG3s3peEdA2LsaHsnZtm3/CAN-Vaccinate?node-id=0%3A1) under "new"). I also updated the feedback form using the link from the [Phase 2 paper doc](https://www.dropbox.com/scl/fi/3iq9uzrtvmjpdi71qd6wq/-Phase-2-Launch-Plan.paper?dl=0&rlkey=gou6giiggng27u8m7eboalc53)

### CODE REVIEW

File: src/components/RegionVaccinationBlock/utils.ts [hunk 1]
```diff
@@ -39,3 +39,8 @@ export const getVaccinationRegions = (region: Region): Region[] => {
 export function trackVaccinationLink(label: string) {
   trackEvent(EventCategory.VACCINATION, EventAction.CLICK_LINK, label);
 }
+
+export const title = 'Getting vaccinated';
+
+export const introText =
+  'Vaccination procedures vary by location. Below are official resources on when and how you can get vaccinated.';
```
Comment (ghop02) [inline src/components/RegionVaccinationBlock/utils.ts:46]: Was there a reason you put this in `utils` as opposed to component?

File: src/components/RegionVaccinationBlock/FeedbackBox.tsx [hunk 2]
```diff
@@ -1,13 +1,27 @@
...
+      </ExternalLink>
     </BoxContainer>
   );
 };
 
+function trackFeedbackClick() {
+  trackEvent(
+    EventCategory.VACCINATION,
+    EventAction.CLICK_LINK,
+    'Suggest and improvement',
```
Comment (ghop02) [inline src/components/RegionVaccinationBlock/FeedbackBox.tsx:23]: *suggest an improvement

**Review by ghop02 — APPROVED** [review]
Looks good! couple small comments

File: src/components/RegionVaccinationBlock/utils.ts (hunk 1 above)
Comment (pnavarrc) [reply src/components/RegionVaccinationBlock/utils.ts:46]: Maybe a component is better, updating!

**Pull request merged**